config/locales/pt-BR.yml

```
@@ -8,3 +8,4 @@ pt-BR:
   welcome_site: Bem Vindo ao nosso sistema de apoio a RPG
```

```suggestion
  links: 
     welcome_site: Bem Vindo ao nosso sistema de apoio a RPG
```

Para chamar vc usa `t(links.welcome_site)`